### ISSUE TITLE
batchnor while using distributed dataparallel

### ISSUE THREAD

**Issue by ParthaEth** [vacancy/Synchronized-BatchNorm-PyTorch#42]
Does this module work if I use distributed data parallel as described here - https://github.com/dougsouza/pytorch-sync-batchnorm-example

**Comment by vacancy** [comment]
No. For distributed data parallel, you can use the PyTorch official implementation: https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html

Note that the official supports only distributed data parallel, and this repo supports only nn.DataParallel
